cate/ds/__init__.py

```diff
@@ -48,5 +48,8 @@ def cate_init():
     from .esa_cci_odp import set_default_data_store
     set_default_data_store()
 
+    from .esa_cci_odp_os import add_data_store
```

Comment:
rename `esa_cci_odp_os` into `esa_cci_odp`

Reply:
Okay. I will consequently add "legacy" from terms that refer to the old data stores and remove "opensearch" to ones that refer to the new one - up to that the identifiers change.